### DISCUSSION TITLE
Accessibility statements in different countries

### DISCUSSION THREAD

**Discussion by sander-nl** [Nordic-Accessibility-Community-Group/en-301-549-resources-and-eaa-monitoring#4] [General]
This thread is meant to provide information about accessibility statements in different (EU) countries. What do they look like, what information is provided, where are they published, etc?

**Comment by erikgustafsson** [comment]
### Links of interest
#### Web-a11y Slack discussions
https://web-a11y.slack.com/archives/CP6NHEEKD/p1728568756046909
https://web-a11y.slack.com/archives/CP6NHEEKD/p1732200095047129

#### Other links
https://www.levelaccess.com/compliance-overview/european-accessibility-act-eaa/
https://accessible.canada.ca/en-301-549-accessibility-requirements-ict-products-and-services-12-documentation-and-support#_Toc66969661
https://arc.net/l/quote/obnaggkp

**Comment by sander-nl** [comment]
Not directly Nordic, but interesting though: in the Netherlands, there is a publicly available dashboard where you can lookup accessibility statements by organisation. See https://dashboard.digitoegankelijk.nl/. 

There is a scoring system:
- A = has published a statement and has an audit report stating that there are no accessibility issues.
- B = has published a statement and has an audit report stating that there are one or more accessibility issues.
- C = has published a statement, but does not have an audit report. An audit is planned within 6 months.
- D = has published a statement, but does not have an audit report and does not have an audit planned. 
- E = has not published a statement.

An audit report, written conform WCAG-EM is mandatory and can be used for 3 years.

**Comment by erikgustafsson** [comment]
Added [accessibility statement template](https://github.com/Nordic-Accessibility-Community-Group/working-with-EN-301-549/blob/main/Accessibility%20Statement%20Template.md) and published [accessibility statement for the EAA article](https://axesslab.com/accessibility-statement-for-the-eaa/).

Finland requires an accessibility statement: [Saavutettavuusvaatimukset](https://www.saavutettavuusvaatimukset.fi/fi/digipalvelulain-vaatimukset/muutokset-digipalvelulakiin)
